test/prob/test_fixture_ccdf_log.hpp

```diff
@@ -129,12 +129,15 @@ class AgradCcdfLogTestFixture : public ::testing::Test {
           << ccdf_log;
 
       if (all_scalar<T0, T1, T2, T3, T4, T5>::value) {
-        EXPECT_TRUE(stan::math::abs(expected_ccdf_log[n] - ccdf_log) < 1e-8)
-            << "For all scalar inputs ccdf_log should match the provided "
-               "value. Failed at index: "
-            << n << std::endl
-            << "expected: " << expected_ccdf_log[n] << std::endl
-            << "actual:   " << ccdf_log;
+        std::stringstream stream;
+        stream << "For all scalar inputs ccdf_log should match the provided "
+                  "value. Failed at index: "
+               << n << std::endl
+               << "expected: " << expected_ccdf_log[n] << std::endl
+               << "actual:   " << ccdf_log;
+
+        stan::test::expect_near_rel(stream.str(), expected_ccdf_log[n],
```

Comment:
I converted all the comparisons to use this comparison function: https://github.com/stan-dev/math/blob/develop/test/unit/math/expect_near_rel.hpp

For various reasons the Google comparisons were annoying to work with. This is the function that is used in the unit tests for this purpose so I use it here too.